src/util/strftime.ts

```diff
@@ -140,7 +140,12 @@ const formatCodes = {
 };
 (formatCodes as any).h = formatCodes.b
 
-export default function (d: Date, formatStr: string) {
+export default function (inputDate: Date, formatStr: string) {
+  let d = inputDate
+  if (d instanceof TimezoneDate && d.timezoneOffset !== null) {
+    d = new Date((+d) + new Date().getTimezoneOffset() * 60 * 1000 + d.timezoneOffset * 60 * 1000)
```

Comment:
And it's better to be `d = d.getDate()` or ` = d.getDisplayDate()`, let TimezoneDate encapsulate the whole complexity.